### ISSUE TITLE
Repositories matching

### ISSUE THREAD

**Issue by spykoulouris** [ArtResearch/vision-app#95]
The system allows user to make the same matching again and again. The reconciliation button appears again for repositories that I have already matched and when I click there it does the matching again. Here the example of the Musee du Petit Palais at Avignon matched twice.

<img width="1785" alt="Screenshot 2022-03-15 at 08 42 55" src="https://user-images.githubusercontent.com/73180833/158321993-d76d0b26-3f68-4699-9d61-5f3609303360.png">

**Comment by mafragias** [comment]
This is not a bug, the two instances from Zeri are seemingly the same, but are actually different, due to the Precise location. This has implemented that way to make the URI of the repository as unique as possible, to avoid mistakes, since there is no ID provided.  (In some cases there is not detailed information about Country/Town or Region, that's why we also take into account the Precise location)

- ![image](https://user-images.githubusercontent.com/23557768/158336681-a0f32570-7cdf-450a-9746-2a96cf5d4929.png)

- ![image](https://user-images.githubusercontent.com/23557768/158336830-ea15d3c8-0b6f-4062-98c2-e95700d4eff4.png)

**Comment by spykoulouris** [comment]
I see, that means that for the Zeri each repository may need to be reconciled multiple (sometimes too many) times?

**Comment by mafragias** [comment]
Yes, but at least, we are certain we don't have merged repositories with the same name!!!!!!!!!!!

**Comment by spykoulouris** [comment]
ok, I understand

**Comment by lklic** [comment]
Hi @spykoulouris  and @mafragias. I think these should be reconciled to the same repository.

The precise location is the inventory number within that location and should be attached to the specific photograph, not the entity of the repository.

We should verify this with Francesca.

Thanks,
L

**Comment by mafragias** [comment]
I do not agree into changing the Repositories' URIs. It will in some cases cause the URIs to merge inconsistently , because it might not have all the other information like country or region and might only have the precise location. Like the example below : 
![image](https://user-images.githubusercontent.com/23557768/158571548-7a58c0c8-88d9-4535-a0ca-b61565132576.png)

**Comment by spykoulouris** [comment]
"Mercato antiquario" and "ubicazione sconosciuta" are not real locations.
I see both benefits and risks here, we should talk with Francesca about this. However, making the reconciliation as it is now makes no sense. Sometimes (i.e. National Gallery of London) we will need to reconcile the same repository hundreds of times.

**Comment by lklic** [comment]
Please provide an extract in csv or link to the spraql query, we can review the data and decide.
